scripts/tts_comparison_report/reporting/__init__.py

```
@@ -0,0 +1,33 @@
+    "DUMMY_TASK_ID",
+    "SUPPORTED_BENCHMARK_NAMES",
+    "TEMPLATES_DIR",
+    "BucketStructure",
+    "Orchestrator",
+    "Renderer",
+    "S3Client",
+    "S3Config",
+    "BaseStorage",
+    "LocalStorage",
+    "SFTPStorage",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)